Fix puppet config steps (#7120)
wazuh/wazuh-documentation: source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-agent.rst

Agent Configuration
^^^^^^^^^^^^^^^^^^^

> Add the server value to the ``[main]`` section of the node ``/etc/puppetlabs/puppet/puppet.conf`` (Linux) or  ``C:\ProgramData\PuppetLabs\puppet\etc\puppet.conf`` (Windows) file, replacing ``puppet-master`` with your Puppet server FQDN if you have your own DNS configured::

Reviewer: ```suggestion
To configure the Puppet agent, edit the configuration file in the node.

-  ``/etc/puppetlabs/puppet/puppet.conf`` for Linux systems
-  ``C:\ProgramData\PuppetLabs\puppet\etc\puppet.conf`` for Windows systems

Add the ``server`` setting to the ``[main]`` section of the file.  If you have set up your own DNS, replace ``puppet-master`` with the Fully Qualified Domain Name (FQDN) of your Puppet server.
```

Author: done

>    [main]

Reviewer: ```suggestion
   .. code-block:: none

      [main]
```

Author: done

>    server = puppet-master

Reviewer: ```suggestion
      server = puppet-master
```

Author: done

Agent Configuration
^^^^^^^^^^^^^^^^^^^

To configure the Puppet agent, edit the configuration file in the node.

> -  ``/etc/puppetlabs/puppet/puppet.conf`` for Linux systems

Reviewer: ```suggestion

-  ``/etc/puppetlabs/puppet/puppet.conf`` for Linux systems
```

-  ``C:\ProgramData\PuppetLabs\puppet\etc\puppet.conf`` for Windows systems

> Add the ``server`` setting to the ``[main]`` section of the file.  If you have set up your own DNS, replace ``puppet-master`` with the Fully Qualified Domain Name (FQDN) of your Puppet server.

Reviewer: ```suggestion

Add the ``server`` setting to the ``[main]`` section of the file.  If you have set up your own DNS, replace ``puppet-master`` with the Fully Qualified Domain Name (FQDN) of your Puppet server.
```
